src/main/client/network/client-buffered-network-component.ts

```diff
@@ -51,7 +51,7 @@ export class ClientBufferedNetworkComponent {
       });
    }
 
-   send(event: NetworkEvent, value: IObject = {}): void {
+   send<T = IObject>(event: NetworkEvent, value: T): void {
```

Comment:
Is this a better practic? Looks redundant. And by doing this "value" does not have a default value and will cause errors.

Reply:
its better to use generics rather than casting with `as` etc

Reply:
ill think of a way to use default value

Reply:
done, default value added

Reply:
Thanks, looks nice!